manifests/certonly.pp

```diff
@@ -62,9 +63,9 @@
   }
 
   if ($custom_plugin) {
-    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly "
+    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly -rsa-key-size ${key_size}"
```

Comment:
Is it really a single dash? I'd expect `--rsa-key-size`

Reply:
Thank you. I am merging in my changes to the module. I missed the - on a copy and paste.